verticapy/tests_new/plotting/highcharts/machine_learning/test_highcharts_model_evaluation.py

```diff
@@ -0,0 +1,343 @@
+"""
+(c)  Copyright  [2018-2023]  OpenText  or one of its
+affiliates.  Licensed  under  the   Apache  License,
+Version 2.0 (the  "License"); You  may  not use this
+file except in compliance with the License.
+
+You may obtain a copy of the License at:
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless  required  by applicable  law or  agreed to in
+writing, software  distributed  under the  License is
+distributed on an  "AS IS" BASIS,  WITHOUT WARRANTIES
+OR CONDITIONS OF ANY KIND, either express or implied.
+See the  License for the specific  language governing
+permissions and limitations under the License.
+"""
+# Pytest
+import pytest
+
+# Standard Python Modules
+
+
+# Other Modules
+
+
+# Verticapy
+from verticapy.learn.ensemble import RandomForestClassifier
+from verticapy.learn.model_selection import lift_chart, prc_curve
+from verticapy.tests_new.plotting.conftest import (
+    get_xaxis_label,
+    get_yaxis_label,
+    get_width,
+    get_height,
+    get_title,
+)
+
+# Testing variables
+COL_NAME_1 = "0"
+COL_NAME_2 = "1"
+BY_COL = "binary"
+POS_LABEL = "0"
+
+
+class TestHighchartsMachineLearningROCPlot:
```

Comment:
I noticed that your test classes leave their trained models behind. The Python object of the model will get automatically garbage collected when all references to the class object have been deleted, but it doesn't drop the model instance in the Vertica DB.
It might not be a big concern for the Unit tests that are executed by our CI/CD tools first because training a new model overwrites a previously trained model with the same name. Besides, a new DB will start each time before running the set of tests.
Nevertheless, it might be annoying for end users who run the unit tests on their local DBs to deal with left over models. To clean up the models, you may add a destructor function to each class.

Reply:
Thanks for pointing that out. I will now be using pytest teardown method to drop models.